CMakeLists.txt

```
@@ -0,0 +1,204 @@
+cmake_minimum_required(VERSION 3.14)
+project(mold VERSION 1.3.0)
+
+include(CMakeDependentOption)
+
+# FIXME: this is for parity with the makefiles that install directly to <prefix>/lib
+# NOTE: defining this before including GNUInstallDirs makes its platform detection a noop
+set(CMAKE_INSTALL_LIBDIR lib CACHE PATH "")
+include(GNUInstallDirs)
+
+# Mold build options
+cmake_dependent_option(MOLD_USE_MIMALLOC "Use mimalloc" ON
+    "NOT CMAKE_SYSTEM_NAME MATCHES Darwin; NOT ANDROID" OFF)
```

```suggestion
    "NOT APPLE;NOT ANDROID" OFF)
```

https://cmake.org/cmake/help/latest/variable/APPLE.html

---

Done, thanks!